core/src/ldml/ldml_processor.cpp

```diff
@@ -390,14 +394,14 @@ ldml_event_state::emit_difference(const std::u32string &old_ctxt, const std::u32
   // marker change.
   // We can detect this because the unchanged_prefix will end with u+FFFF U+0008
   //
-  // Oh, and yes, test case 'regex-test-8a-0' hits this.
```

Comment:
what an embarrasingly proud comment :( 

That test case _used_ to hit this.  But, I added `8a` as a prefix in the output string to distinguish it from the other cases in k_009.  

For this fix I took a different tack:
- 1 completely separate isolated file (k_212) with the regression test.
- 1 original-plus-embedded-markup file (k_213) as submitted by the [Affected User](https://github.com/srl295) 

Yes, it makes meson take longer to compile. But now there's a more stable regression test.